src/libs/AxelarQueryAPI.ts

```diff
@@ -308,4 +314,73 @@ export class AxelarQueryAPI {
       });
     }
   };
+
+  public async getTransferLimit({
+    chainId,
+    denom,
+  }: {
+    denom: string;
+    chainId: string;
+  }): Promise<string> {
```

Comment:
I would rather avoid using an object as a function parameter in this class to keep it consistent with other public APIs.

Reply:
i think we decided to change everything to objects as function parameters. will gradually migrate everything to use this instead. part of the reason is that it'll become easier to manage/update in the future if we need.